### PR TITLE
feature/FOUR-8243

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
log a Request error

## How to Test
- Create a Request with an error
- Run the request

## Related Tickets & Packages
[- Link to any related FOUR tickets, PRDs, or packages
](https://processmaker.atlassian.net/browse/FOUR-8243)

## Code Review Checklist
- [x] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [x] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [x] This solution fixes the bug reported in the original ticket.
- [x] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [x] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [x] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [x] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Enviroment
ci:deploy

### CODE REVIEW

File: ProcessMaker/Events/RequestError.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        $this->data = $data;
+        $this->error = $error;
+    }
+    
+    /**
+     * Return event data 
+     */
+    public function getData(): array
+    {
+        $parentProcess = Process::find($this->data->getAttribute('process_id'));
```
Comment (fagubla) [inline ProcessMaker/Events/RequestError.php:34]: delete this variable is not being used

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

File: ProcessMaker/Models/ProcessRequest.php [hunk 2]
```diff
@@ -552,6 +553,7 @@ public function logError(Throwable $exception, FlowElementInterface $element = n
         \Log::error($exception);
```
Comment (caleeli) [inline ProcessMaker/Models/ProcessRequest.php:553]: If this `\Log::error()` is not required or was used for debugging, please remove it.

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-8d49dc98ff.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**